README.md

```diff
@@ -23,9 +23,15 @@ Features of `react-image-gallery`
 ## Getting started
 
 ```
+npm install react
```

Comment:
This can be removed

Reply:
See https://github.com/xiaolin/react-image-gallery/pull/434#discussion_r314672521